packages/web-app-admin-settings/src/views/Spaces.vue

```diff
@@ -155,6 +158,41 @@ export default defineComponent({
       toggleSelectSpace,
       unselectAllSpaces
     }
+  },
+  computed: {
```

Comment:
Please use the Composition API. For computed props, you can use `computed(() => { ... })` (like done above) and methods can be simply written as arrow-functions.

In general, we want to use the Composition API in the future because it's the recommended way in Vue 3.

Reply:
sí 👍🏼